TicketBookingFrontend/src/app/models/bus.model.ts

```diff
@@ -0,0 +1,6 @@
+export class BusModel
```

Comment:
Change the name to bus instead of busmodel

Reply:
changed